canonical-web-and-design/ubuntu.com
# 2269 update raspberry pi page

## Done

- Fixup release variables and logic for raspberry pi buttons, thankyou and verification sections
- Add links to tutorials

## QA

- Download this branch
- ./run
- look at http://0.0.0.0:8001/download/raspberry-pi/
- select some images and test http://0.0.0.0:8001/download/raspberry-pi/thank-you and with some get parameters (from clicking a button)
- See that the downloads start, see that the verify instructions work
- Test the links on the page.

Fixes #2269 and #2300
